src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadataSettings.cs

```diff
@@ -19,17 +19,21 @@ public class XbmcMetadataSettings : IProviderConfig
         public XbmcMetadataSettings()
         {
             MovieMetadata = true;
+            MovieMetadataURL = true;
             MovieImages = true;
             UseMovieNfo = false;
         }
 
         [FieldDefinition(0, Label = "Movie Metadata", Type = FieldType.Checkbox)]
         public bool MovieMetadata { get; set; }
 
-        [FieldDefinition(1, Label = "Movie Images", Type = FieldType.Checkbox)]
+        [FieldDefinition(1, Label = "Movie Metadata URL", Type = FieldType.Checkbox, HelpText = "Radarr will write the tmdb/imdb url in the .nfo file")]
```

Comment:
I thinks this should be moved downwards

Reply:
I would rather keep them together, since they concern the same file; but if you really feel like it I can certainly move those down